ui/appui-abstract/src/appui-abstract/BaseUiItemsProvider.ts

```diff
@@ -16,9 +16,9 @@ import { StageUsage } from "./items/StageUsage";
 import { UiItemsProvider } from "./UiItemsProvider";
 import { UiItemsManager } from "./UiItemsManager";
 
-/**
- * Base implementation of a UiItemsProvider. The base class allows the user to pass in a function that is used to determine if the
+/** Base implementation of a UiItemsProvider. The base class allows the user to pass in a function that is used to determine if the
  * active stage should be provided items. Derived provider classes should override the `xxxInternal` methods to provide items.
+ * @deprecated in 3.6. Use [BaseUiItemsProvider]($appui-react) instead.
```

Comment:
is this an api we really want to bring over? thought this was a temp a solution for DR that we said we'd fix for 4.0

Reply:
Not sure if we will manage to achieve that in time for 4.0. Since the type is marked as public we have to provide a replacement by moving it.. This will allow us to tackle the API problem separately in `appui` repository.

Reply:
yea timing wise, im concerned we'll be able to clean up any of the api in time
imo this should have probably never been public, it was introduced in 3.0 too
should chat later if we always need a replacement when we do a deprecation, what about just removals in general

Reply:
Agree with Arun: this was a solution for DR that has since been superseded and needs to go.